apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -669,6 +671,15 @@ export class ApiModelGenerator {
     this._processChildDeclarations(astDeclaration, exportedName, apiInterface);
   }
 
+  private _determineReadonly(astDeclaration: AstDeclaration): boolean {
```

Comment:
Are we handling the `const SOMETHING = 5` case yet?

Reply:
No, will look into adding now.

Reply:
Added case